backend/dal/Migrations/v01.07.02/Up/PostUp/Adminstrative_Areas_And_Addresses.sql

```diff
@@ -0,0 +1,191 @@
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Lake Cowichan' 
+WHERE AdministrativeArea = 'Town of Lake Cowichan';
+
+UPDATE [dbo].[Addresses]
+SET AdministrativeArea = 'Kent' 
+WHERE AdministrativeArea = 'District of Kent';
+
+
+PRINT 'Updating AdministrativeAreas table'
+
+-- Then update AdministrativeAreas table to fix the Abbreviation field in order to get correct join when doing a batch update later on
+UPDATE [dbo].[AdministrativeAreas]
+SET Abbreviation = 'Valemount' 
+WHERE Name = 'Village of Valemount' AND Abbreviation = 'Valemont';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Abbreviation = 'Fort St. John' 
+WHERE Name = 'City of Fort St John' AND Abbreviation = 'Fort St John';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Abbreviation = 'Fort St. James' 
+WHERE Name = 'District of Fort St James' AND Abbreviation = 'Fort St James';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Abbreviation = 'North Vancouver' 
+WHERE Name = 'City of North Vancouver' AND Abbreviation = 'North Vancouver - City';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Abbreviation = 'Langley' 
+WHERE Name = 'City of Langley' AND Abbreviation = 'Langley - City';
+
+-- Do the "one off" updates to simplify the name where there is only once instance of each
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Spallumcheen' 
+WHERE Name = 'The Corporation of the Township of Spallumcheen';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Lake Country' 
+WHERE Name = 'District Of Lake Country';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Port Edward' 
+WHERE Name = 'District of Port Edward';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Highlands' 
+WHERE Name = 'District of Highlands';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Belcarra' 
+WHERE Name = 'Village of Belcarra';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Slocan' 
+WHERE Name = 'Village of Slocan';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Anmore' 
+WHERE Name = 'Village of Anmore';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Warfield' 
+WHERE Name = 'Village of Warfield';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Canal Flats' 
+WHERE Name = 'Village of Canal Flats';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Radium Hot Springs' 
+WHERE Name = 'Village of Radium Hot Springs';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'North Cowichan' 
+WHERE Name = 'The Corporation of the District of North Cowichan';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Coldstream' 
+WHERE Name = 'The Corporation of the District of Coldstream';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Lake Cowichan' 
+WHERE Name = 'Town of Lake Cowichan';
+
+UPDATE [dbo].[AdministrativeAreas]
+SET Name = 'Kent' 
+WHERE Name = 'District of Kent';
+
+-- Then update GroupName field for existing locations whose name is the same as the Abbreviation field (records from BC Data) before deleting the records from BC Data
+UPDATE t1
```

Comment:
It looks like you're essentially just copying the `GroupName` from the `AdministrativeArea` that has a `Name = Abbreviation`, but this is hard to read.